src/Sulu/Component/Rest/ListBuilder/Expression/ConjunctionExpressionInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Expression;
+
+/**
+ * Interfaces which provides the needed information to build a conjunction-expression.
+ */
+interface ConjunctionExpressionInterface extends ExpressionInterface
+{
+    /**
+     * @return string
+     */
+    public function getConjunction();
```

Comment:
I would remove this method, since it is not needed at all, I also couldn't find any use cases, and I also can't imagine any.